swift/Workflow/Sources/SubtreeManager.swift

```diff
@@ -204,7 +204,11 @@ extension WorkflowNode.SubtreeManager {
         func makeSink<Action>(of actionType: Action.Type) -> Sink<Action> where Action: WorkflowAction, WorkflowType == Action.WorkflowType {
             let reusableSink = sinkStore.findOrCreate(actionType: Action.self)
 
+            let signpostRef = NSObject()
```

Comment:
Wonder if we should create a type which has the `log...` events and acts as a `ref` instead of creating an arbitrary object to act as `ref`.

Reply:
🤷 I'm not overly-concerned about it. We could make the `log...` calls member functions instead of static functions (and use the object as the object we use to look up the signpost ID. We'd have to figure out how to plumb that object around to the various places that need it.

I think this is fine for now. Thoughts?

Reply:
I meant, creating something like:

```swift
class SignpostRef: AnyObject {
  func log(...)
}
```

But, this is just a nit, feel free to ship as-is!